README.md

```diff
@@ -18,99 +22,137 @@ We have active, helpful communities on Twitter, Discord, and Telegram.
+    && sudo mv sommelier.service /etc/systemd/system/ \
+    && sudo systemctl daemon-reload
+
+# Init steward/orchestrator configuration. Note that the steward and orchestrator processes share
+# much of the same configuration fields, so we share the config.toml for convenience.
+mkdir -p $HOME/steward && cd $HOME/steward
 wget https://raw.githubusercontent.com/PeggyJV/sommelier/main/contrib/mainnet/sommelier-3/config.toml
 
-# modify gorc config for your environment
-# You can use alchemy, infura endpoint as RPC and not necessarily need to sync the blockchain eth with geth
+# modify steward/orchestrator config for your environment, in particular your RPC URL and keystore path
 nano config.toml
 
 # Initialize the validator files
 sommelier init myval --chain-id sommelier-3
+```
 
-# create/restore 2 cosmos keys and 1 ethereum key
-# NOTE: be sure to save the mnemonics and eth private key
-
-# restore orchestrator key with gorc 
-gorc --config $HOME/gorc/config.toml keys cosmos recover orchestrator "{menmonic}"
-
-# OR: create orchestrator key with gorc 
-gorc --config $HOME/gorc/config.toml keys cosmos add orchestrator
-
-# restore eth key 
-
-# EITHER: restore eth priv key from metamask with gorc 
-gorc --config $HOME/gorc/config.toml keys eth import signer "0x0000..."
-
-# OR: restore eth mnemonic with gorc
-gorc --config $HOME/gorc/config.toml keys eth recover signer "{menomonic}"
-
-# OR: create eth key with gorc 
-gorc --config $HOME/gorc/config.toml keys eth add signer
+At this point you need to create orchestrator keys OR restore them if you
+already created them. __Please follow [these
+instructions](https://github.com/PeggyJV/steward/blob/main/docs/03-TheOrchestrator.md#setup)
+to create or restore these keys with the Steward CLI__, then return to this doc
+for steps to add them to your validator.
 
+```bash
 # restore your validator mnemonic to the sommelier binary
 sommelier keys add validator --recover
 
 # OR: create your validator mnemonic to the sommelier binary
 sommelier keys add validator
 
 # NOTE: at the end of this process you need to have:
-# - a key named "orchestrator" with funds on the cosmos chain in the gorc keystore
-# - a key named "signer" with funds on connected ETH chain in the gorc keystore
+# - a key named "orchestrator" with funds on the cosmos chain in the steward keystore
+# - a key named "signer" with funds on connected ETH chain in the steward keystore
 # - a key named "validator" with funds on the cosmos chain in the sommelier keystore
 
 # Add the peers from contrib/mainnet/sommelier-3/peers.txt to the ~/.sommelier/config/config.toml file
 nano ~/.sommelier/config/config.toml
 
-# pull the genesis file 
-wget https://raw.githubusercontent.com/PeggyJV/sommelier/main/contrib/mainnet/sommelier-3/genesis.json -O $HOME/.sommelier/config/genesis.json
+# pull the genesis file
+wget https://raw.githubusercontent.com/PeggyJV/sommelier/main/contrib/mainnet/sommelier-3/genesis.json \
```

Comment:
Is there some missing context that places `genesis.json` in the correct place?

Reply:
next line is the output, there's a `\` line break